guide/payments/send.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: guide
+title: Sending bitcoin
+description: An introduction to how sending bitcoin works.
+nav_order: 2
+parent: Payments
+permalink: /guide/payments/send/
+main_classes: -no-top-padding
+image: /assets/images/guide/payments/overview/page-overview.svg
+---
+
+# Sending bitcoin
+
+Sending bitcoin can be a very straight forward or complex flow in a bitcoin application. Users may be sending bitcoin to a known contact, moving it between their own wallets on different devices, or making a purchase through a [payment processor](https://bitcoin.design/guide/getting-started/software/#payment-processors).
+
+Regardless of where the bitcoin is being sent, there are some basic configurations that an application needs to help the user with when creating a transaction: 
+
+---
+
+- **Recipient address** — Where to send the bitcoin
+- **Amount** — How much to send
+- **Coin selection** — Which coins/inputs to use (optional)
+- **Fee settings** — Prioritize fast confirmation or low cost (optional)
+
+---
+
+The order of configuration for the payment can be tailored to your use case. For example, you may make users set the amount before the address is entered.
+            
+## Get the recipient address
+To send a payment on the Bitcoin blockchain we need the recipient's address. Since Bitcoin [addresses](https://bitcoin.design/guide/glossary/#address) are long and seemingly random, they are best shared by copying and pasting in plain text, as a [payment link](https://bitcoin.design/guide/foundations/wallet-interoperability/#payment-links), or as a scannable [QR Code](https://bitcoin.design/guide/foundations/wallet-interoperability/#qr-codes).
+
+This is easily accomplished by generating a new address in the receiving wallet application, then sharing it with the sender. If the sender and receiver are together at the time, scanning the receivers address as a QR code will be easy, but if they are not, they can send the address as text in any regular communication tool like email, sms etc.
+
+## Input an address
+
+Once you have gotten the address, it needs to be inputted on the "create transaction" interface. Bitcoin transactions are irreversible so both the sender and receiver should take great care in correctly sharing and inputting addresses. 
+
+Upon address input, the user should be clearly informed if the address is valid or not. If the address is not valid, or incompatible with the current application, sending should be disabled.
+
+**QR Code** -- Once the camera detects a valid address in the QR Code, it can be automatically inputted. The user will need to grant access your application to be able to scan QR Codes.
```

Comment:
```suggestion
**QR Code** -- Once the camera detects a valid address in the QR Code, it can be automatically inputted. The user will need to grant access to your application to be able to scan QR Codes.
```